python/pyiceberg/table/sorting.py

```diff
@@ -36,11 +36,17 @@ class SortDirection(Enum):
     ASC = "asc"
     DESC = "desc"
 
+    def __str__(self):
+        return str(self.name)
+
 
 class NullOrder(Enum):
     NULLS_FIRST = "nulls-first"
     NULLS_LAST = "nulls-last"
 
+    def __str__(self):
+        return str(self.name)
```

Comment:
Do we want this to be the SQL sort order? If so, we probably want to remove the `_` or `-` in the name or value. Otherwise we'll get output like `day(ts) DESC NULLS_LAST`.

Reply:
Yes, I think we do want that. I'll make this equivalent to Java: https://github.com/apache/iceberg/blob/master/api/src/main/java/org/apache/iceberg/NullOrder.java